src/annotator/components/toolbar.js

```diff
@@ -187,13 +202,94 @@ export default function Toolbar({
             <ToolbarButton
               label="Pen"
               icon="pen"
-              onClick={() => setDoodleOptions({ tool: 'pen', size: 7 })}
+              onClick={() => {
+                setTool('pen');
+              }}
             />
             <ToolbarButton
               label="Eraser"
               icon="erase"
-              onClick={() => setDoodleOptions({ tool: 'eraser', size: 25 })}
+              onClick={() => {
+                setTool('eraser');
+              }}
             />
+            <button
+              className="popup annotator-toolbar-button size selected"
+              onClick={e => {
+                //@ts-ignore
+                e.target?.querySelector('#myPopup').classList.toggle('show');
+                //@ts-ignore
+                e.target?.parentElement
+                  .querySelector('.color')
+                  .querySelector('#myPopup')
+                  .classList.remove('show');
+              }}
+              aria-label="Brush Size"
+            >
+              <SvgIcon name="sizes-icon" className="svgicon" />
+              <span className="popuptext" id="myPopup">
```

Comment:
I think that element IDs are supposed to be unique across the DOM. Maybe we could rename one to be "mySizePopup" and one to be "myColorPopup"? Or, if these elements are going to be changed up when we switch to the different type of toolbar, they're fine as they are.